ui/pages/GasTracker.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Flex, Skeleton, chakra } from '@chakra-ui/react';
+import React from 'react';
+
+import config from 'configs/app';
+import useApiQuery from 'lib/api/useApiQuery';
+import dayjs from 'lib/date/dayjs';
+import { HOMEPAGE_STATS } from 'stubs/stats';
+import GasTrackerChart from 'ui/gasTracker/GasTrackerChart';
+import GasTrackerNetworkUtilization from 'ui/gasTracker/GasTrackerNetworkUtilization';
+import GasTrackerPrices from 'ui/gasTracker/GasTrackerPrices';
+import GasInfoUpdateTimer from 'ui/shared/gas/GasInfoUpdateTimer';
+import PageTitle from 'ui/shared/Page/PageTitle';
+
+const GasTracker = () => {
+  const { data, isPlaceholderData, isError, error, dataUpdatedAt } = useApiQuery('homepage_stats', {
```

Comment:
maybe rename it to 'stats' or something? since it's not only for homepage anymore